modin/pandas/indexing.py

```diff
@@ -82,84 +82,68 @@ def compute_sliced_len(slc, sequence_len):
 
 def is_2d(x):
     """
-    Implement [METHOD_NAME].
-
-    TODO: Add more details for this docstring template.
+    Check that argument is a list or a slice.
 
     Parameters
     ----------
-    What arguments does this function have.
-    [
-    PARAMETER_NAME: PARAMETERS TYPES
-        Description.
-    ]
+    x : object
+        Object to check.
 
     Returns
     -------
-    What this returns (if anything)
+    bool
+        `True` if argument is a list or slice, `False` otherwise.
     """
     return is_list_like(x) or is_slice(x)
 
 
 def is_tuple(x):
```

Comment:
and this function looks redundant IMHO

Reply:
It is not a part of documentation tasks to change the code, isn't it?

Reply:
right, but it might be worth it putting a `FIXME`